desci-contracts/scripts/deployDpidAliasRegistry.js

```diff
@@ -0,0 +1,31 @@
+const { ethers, upgrades } = require("hardhat");
+const fs = require('fs');
+
+const FIRST_DPID = 500;
```

Comment:
Why can't we pause the contract and then get FIRST_DPID value from the paused contract. So if the last dpid is 250, make FIRST_DPID = 251 or similar